### PR TITLE
Add 'go give a start on github' on start page

### PR DESCRIPTION
**Pull request by maxandersen**
a bunch of us watching jhipster demo thought it could be interesting to add a "give star" on main index page as they do.

This is my proposal on how that could look like.

### CODE REVIEW

**Comment by geoand** [comment]
Strictly speaking, only the `index.ftl` file change is needed

**Comment by maxandersen** [comment]
just fearing smoe tests would fail if output wasn't properly matched and by doing a regular expression replacement i could ensure all was updated wither .ftl or .html

**Comment by geoand** [comment]
Yeah, it's not a problem at all

**Review by geoand — APPROVED** [review]

**Review by gastaldi — APPROVED** [review]

File: independent-projects/tools/common/src/test/resources/templates/index.ftl [hunk 1]
```diff
@@ -123,6 +123,9 @@
             </li>
         </ul>
 
+        <h2>Do you like Quarkus ?</h2>
+        <p>Go give it a star on <a href="https://github.com/quarkusio/quarkus">github</a>.</p>
```
Comment (gastaldi) [inline independent-projects/tools/common/src/test/resources/templates/index.ftl:127]:
```suggestion
        <p>Go give it a star on <a href="https://github.com/quarkusio/quarkus">GitHub</a>.</p>
```

**Comment by gastaldi** [comment]
Looks good, but I'd go with `GitHub`, not `github` :)

File: devtools/gradle/src/functionalTest/resources/add-extension-multi-module/application/src/main/resources/META-INF/resources/index.html [hunk 2]
```diff
@@ -123,6 +123,9 @@ <h2>What can I do from here?</h2>
             </li>
         </ul>
 
+        <h2>Do you like Quarkus ?</h2>
```
Comment (gsmet) [inline devtools/gradle/src/functionalTest/resources/add-extension-multi-module/application/src/main/resources/META-INF/resources/index.html:126]:
```suggestion
        <h2>Do you like Quarkus?</h2>
```

File: devtools/gradle/src/functionalTest/resources/basic-java-library-module/application/src/main/resources/META-INF/resources/index.html [hunk 3]
```diff
@@ -123,6 +123,9 @@ <h2>What can I do from here?</h2>
             </li>
         </ul>
 
+        <h2>Do you like Quarkus ?</h2>
```
Comment (gsmet) [inline devtools/gradle/src/functionalTest/resources/basic-java-library-module/application/src/main/resources/META-INF/resources/index.html:126]:
```suggestion
        <h2>Do you like Quarkus?</h2>
```

File: devtools/gradle/src/functionalTest/resources/basic-multi-module-project/application/src/main/resources/META-INF/resources/index.html [hunk 4]
```diff
@@ -123,6 +123,9 @@ <h2>What can I do from here?</h2>
             </li>
         </ul>
 
+        <h2>Do you like Quarkus ?</h2>
```
Comment (gsmet) [inline devtools/gradle/src/functionalTest/resources/basic-multi-module-project/application/src/main/resources/META-INF/resources/index.html:126]:
```suggestion
        <h2>Do you like Quarkus?</h2>
```

File: devtools/platform-descriptor-json/src/main/resources/templates/index.ftl [hunk 5]
```diff
@@ -123,6 +123,9 @@
             </li>
         </ul>
 
+        <h2>Do you like Quarkus ?</h2>
```
Comment (gsmet) [inline devtools/platform-descriptor-json/src/main/resources/templates/index.ftl:126]:
```suggestion
        <h2>Do you like Quarkus?</h2>
```

File: independent-projects/tools/common/src/test/resources/templates/index.ftl [hunk 6]
```diff
@@ -123,6 +123,9 @@
             </li>
         </ul>
 
+        <h2>Do you like Quarkus ?</h2>
```
Comment (gsmet) [inline independent-projects/tools/common/src/test/resources/templates/index.ftl:126]:
```suggestion
        <h2>Do you like Quarkus?</h2>
```

File: integration-tests/kotlin/src/test/resources/projects/classic-kotlin/src/main/resources/META-INF/resources/index.html [hunk 7]
```diff
@@ -123,6 +123,9 @@ <h2>What can I do from here?</h2>
             </li>
         </ul>
 
+        <h2>Do you like Quarkus ?</h2>
```
Comment (gsmet) [inline integration-tests/kotlin/src/test/resources/projects/classic-kotlin/src/main/resources/META-INF/resources/index.html:126]:
```suggestion
        <h2>Do you like Quarkus?</h2>
```

**Review by gsmet — CHANGES_REQUESTED** [review]
See @gastaldi comment and mine. Those should be applied everywhere.

Also I would like @emmanuelbernard 's blessing on this type of things.

**Comment by maxandersen** [comment]
okey thats just weird - I pushed master to this fork and github decides to close it and now can't reopen it....guess I'll have to make another pullreuqest then

**Pull request closed without merging**